### PR TITLE
Cleanup warnings in noapi PlatformActivity

### PR DESCRIPTION
**Pull request by bluemarvin**

### CODE REVIEW

File: app/src/noapi/java/org/mozilla/vrbrowser/PlatformActivity.java [hunk 1]
```diff
@@ -252,6 +241,7 @@ private void setupUI() {
             }
             return false;
         });
+        setImmersiveSticky();
```
Comment (keianhzo) [inline app/src/noapi/java/org/mozilla/vrbrowser/PlatformActivity.java]: Why is this required now? Android 10?

Comment (bluemarvin) [reply app/src/noapi/java/org/mozilla/vrbrowser/PlatformActivity.java]: It seems to lose it some times on my phone so I added it a few more place to try and reset it for when it gets unset.
